dev-libs/spdlog/Manifest

```diff
@@ -1,2 +0,0 @@
-DIST spdlog-0.16.2.tar.gz 162709 BLAKE2B 7cd031bd9b1e01aa11186334ada9aad50cdc063942e0eb5b9d36e918e305db4c0e5657a215f24f458dbf792e05c38f74409bc7f5f88049bd1c4a759ccab0167e SHA512 454da17d75fa6dae074fb2ef10d93a1f9ccbf23b625521d32ce463a1362411ffc35ca33b081bee3cfb27bb647472de73899c9a15d24f09c8a3c9222560bcf559
```

Comment:
Technically, this belongs after adding new version. Otherwise you introduce an interim state where there are no ebuilds here (which would be bad e.g. when bisecting).

Reply:
Play with `git rebase -i origin/master`. That's probably the single most powerful tool inside git.

Reply:
You're removing DIST entries too early.